packages/owl-bot/src/database.ts

```diff
@@ -161,7 +161,9 @@ export class FirestoreConfigsStore implements ConfigsStore, CopyTasksStore {
     // to optimize if performance becomes a problem.
     const snapshot = await this.db.collection(this.yamls).get();
     const result: string[] = [];
+    let i = 0;
     snapshot.forEach(doc => {
+      i++;
       const configs = doc.data() as Configs | undefined;
       match_loop: for (const copy of configs?.yaml?.['copy-dirs'] ?? []) {
         const mm = newMinimatchFromSource(copy.source);
```

Comment:
Oh dear.  This code should be using regular expressions instead of minimatch.  I'll prepare a PR.

Reply:
Nevermind.  The code that switches this to regular expressions is in my pending PR.